web/sites/default/production.services.yml

```diff
@@ -1,11 +0,0 @@
-### amazee.io Drupal 8 production environment services file.
```

Comment:
previously, it was possible to modify any of the setting files within a project. Now, when these files are removed, how would one can override something per-environment?